### ISSUE TITLE
Add Edit, Pagination, and export handling-options for multi-statement queries 

### ISSUE THREAD

**Issue by stokkeland** [phpmyadmin/phpmyadmin#17051]
### Is your feature request related to a problem? Please describe.

Consider these two statements:
```
SET @num:=0;
SELECT COALESCE(@num:=@num+1,'Uservar Failed') AS TestCol
from any_table
limit 5
```
When First executed in PHPMyadmin, they produce the 1,2,3,4,5 rows. If you click export, it will produce rows of "Uservar Failed", because the first statement was not executed. Similar scenarios will happen with Pagination if you use a user variable set like that for an actual table query.
The functionality i have found that breaks is Export, Pagination, and Edit (There is no way to get back to edit what was submitted as a whole).

### Describe the solution you'd like
When a query with multiple statements is submitted, provide some additional options, such as
- Link/Button for Edit Original Query as it was Written
- Checkbox or Radio that allows "Execute all N queries" which triggers it for Export and Pagination
(or some other clever way to allow the same)

### Describe alternatives you've considered
(I could Hard type/expand queries to not need user variables that need to be set before select statement, or use non sql scripting outside phpmyadmin).
The biggest annoyance is the inability to go back and edit full query exactly as it was submitted, 

### Additional context

- phpMyAdmin version: 4.9.5deb2 (Ubuntu 20LTS)

**Comment by williamdes** [comment]
Hi @stokkeland 

Thank you for this report, it seems linked to #12156

> phpMyAdmin version: 4.9.5deb2 (Ubuntu 20LTS)

You might want to use our PPA to upgrade to 5.1.1 using the [wiki guide](https://github.com/phpmyadmin/phpmyadmin/wiki/DebianUbuntu#ubuntu-ppa)

**Comment by williamdes** [comment]
> Link/Button for Edit Original Query as it was Written

On 5.1 there is a query box

![image](https://user-images.githubusercontent.com/7784660/128617221-af3defff-04e3-4b07-8768-1985960b1a41.png)

> The biggest annoyance is the inability to go back and edit full query exactly as it was submitted,

Seems like it would be some flavor of #12117

**Comment by stokkeland** [comment]
You are right, the show/hide query box that came in 5.1 does relieve my "biggest annoyance" of editing the original submit.

And the mentioned/linked issues do appear to be flavors/overlap of this, i didn't really look at it like bugs, i thought more about that an enhancement could be the ability to handle "all statements" every time an action is taken, like export and pagination. feel free to close this if you think it is sufficiently covered in the other issues.

**Comment by williamdes** [comment]
I think it is good to be kept open as you documented very well the issue and the duplicated one is quite old. In the closing process the older one would probably win for the changelog entry

But I agree an edit or go to sql option could be a nice to have
